### DISCUSSION TITLE
Who wrote the tweet? With StreamListener

### DISCUSSION THREAD

**Discussion by Tromds** [tweepy/tweepy#1636] [Questions] [answered]
**Hi, Tweepy is a great project and I'm in love.** 

I have created a script that searches for all the tweets that are written from my city, Benidorm. So I can read all tweets and retweets that have the word Benidorm in them.

Now I would like to know the name of the user who wrote the tweet. But I can't find the solution. 

Is it possible to have the tweet next to the user who posted the tweet?

Thank you very much!

Translated with www.DeepL.com/Translator (free version)

**Comment by Harmon758** [comment, marked as answer]
> Now I would like to know the name of the user who wrote the tweet. But I can't find the solution.

If you have the Status/[Tweet object](https://developer.twitter.com/en/docs/twitter-api/v1/data-dictionary/object-model/tweet), you can use its `user` attribute to retrieve the [User object](https://developer.twitter.com/en/docs/twitter-api/v1/data-dictionary/object-model/user) representing the user who posted the Tweet. You can then use the `name` attribute of that User object.

> Is it possible to have the tweet next to the user who posted the tweet?

I'm not sure what you mean by "next to".
